managed-services/iot-hub/reference-content/hubs.mdx

```diff
@@ -36,7 +36,7 @@ There are three IoT Hub plans available:
   A table describing the features of each plan in detail is available in the [Hub creation wizard](https://console.scaleway.com/iot-hub/hubs/create) on the console.
 </Message>
 
-## Certificate Authority
+## Certificate authority
```

Comment:
Are you sure about the lowercase? cf https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/api-cli/verify-ca-postgresql

Reply:
Both are possible - but ok
```suggestion
## Certificate Authority
```